app/src/main/java/Fragments/SettingsFragment.java

```diff
@@ -52,18 +54,31 @@ public void onStart() {
 
         logInB = (Button)getView().findViewById(R.id.logInB);
         logOutB = (Button)getView().findViewById(R.id.logOutB);
+        signUpB = (Button)getView().findViewById(R.id.signUpB);
+        aboutB = (Button)getView().findViewById(R.id.aboutB);
+        notificationsB = (Button)getView().findViewById(R.id.notificationsB);
+
+
+
 
         // SET Auth State Listener
         authListener = new FirebaseAuth.AuthStateListener() {
             @Override
             public void onAuthStateChanged(@NonNull FirebaseAuth firebaseAuth) {
                 if(authRef.getCurrentUser() != null) {
```

Comment:
Minor tweak suggestion: I would add for the if statement : user logged in 
and for the else statement: user logged out, just for reference purposes!

Reply:
Can this be implemented in a future sprint.